timbermill-local/src/main/java/com/datorama/oss/timbermill/common/ElasticsearchUtil.java

```diff
@@ -28,6 +26,9 @@
 
 public class ElasticsearchUtil {
 	public static final String CLIENT = "client";
+	public static final String DISK_HANDLER = "disk_handler";
+	public static final String EVENTS_QUEUE = "events_queue";
+	public static final String OVERFLOWED_EVENTS_QUEUE = "overflowed_events_queue";
 	public static final int THREAD_SLEEP = 2000;
 	public static final String SCRIPT =
```

Comment:
what do you say about moving it to resources dir and read from there?

Reply:
why?

Reply:
it makes this class huge and ugly

Reply:
I kinda like it here. Without the hassle of reading a file, writing a file.